src/Sylius/Behat/Context/Setup/ShippingContext.php

```diff
@@ -122,13 +123,15 @@ public function shippingMethodBelongsToTaxCategory(ShippingMethodInterface $ship
 
     /**
      * @param string $name
+     * @param string $code
      * @param ZoneInterface|null $zone
      * @param string $locale
      * @param array $configuration
      * @param string $calculator
      */
     private function createShippingMethod(
         $name,
+        $code,
```

Comment:
If every argument after ``$code`` is optional, can't we make it optional too - ``$code = null``?